README.md

```diff
@@ -563,21 +563,48 @@ default in a reduced functionality ("lite") mode that does not require
 PostgreSQL. To enable the PostgreSQL backend (and the expanded functionality),
 dcrdata may be started with the `--pg` switch.
 
+## APIs
+
+The dcrdata block explorer is exposed by two APIs: a Decred implementation of the [Insight API](https://github.com/bitpay/insight-api) (EXPERIMENTAL), and its own JSON HTTP API. The Insight API uses the path prefix `/insight/api`. The dcrdata API uses the path prefix `/api`.
+
 ### Insight API (EXPERIMENTAL)
 
-The [Insight API](https://github.com/bitpay/insight-api) is accessible via HTTP
-at the `/insight/api` URL path prefix, and via WebSocket at the
-`/insight/socket.io` URL path prefix.
+The [Insight API](https://github.com/bitpay/insight-api) is accessible via HTTP via REST or WebSocket. 
+
+To call the REST API, use the `/insight/api` path prefix. To call the Websocket API, use the `/insight/socket.io` path prefix.
 
 The following endpoints are not implemented: `status`, `sync`, `peer`, `email`,
 and `rates`.
 
+#### Endpoint List
+
+| Method           | Path                   | 
+| -------------------- | ---------------------- | 
+| Block              | `/block/{hash}`          |  
+| Block index          | `/block-index/{height}`      |  
+| Raw block (hash)               | `/rawblock/{hash}`   | 
+| Raw block (height)                | `/rawblock/{height}`     |  
+| Raw summaries               | `/blocks (URL query params: limit, blockDate, ...) `   |              
+| Transaction                 | `/tx/{hash}`     |  
+| Raw transaction              | `/rawtx/{hash}`  | 
+| Address         | `/addr/{address}`       | 
+| Address Props   | `/balance, totalReceived, totalSent, /totalSent, unconfirmedBalance` |  
+| UTXOs | `/addr/{address}/utxo`  |  
+| Multi-address UTXOs | `/addrs/{address0, address1, etc}/utxo`  |  
+| POST-variant of multi-address UTXOs | ``  |                     
```

Comment:
For the optional parameters in the case of POST endpoints we should display them slightly differently since they need to be posted in the JSON body.  in this case:

**Required**:  `{"addrs":"[ADDR1],[ADDR2]"}`